src/webui/package.json

```diff
@@ -22,12 +22,16 @@
     "html-webpack-plugin": "4.0.0-beta.5",
     "identity-obj-proxy": "3.0.0",
     "is-wsl": "^1.1.0",
+    "jest": "24.7.1",
+    "jest-environment-jsdom-fourteen": "0.1.0",
+    "jest-resolve": "24.7.1",
+    "jest-watch-typeahead": "0.3.0",
     "json5": "^2.1.1",
     "mini-css-extract-plugin": "0.5.0",
     "monaco-editor": "^0.19.0",
     "monaco-editor-webpack-plugin": "^1.8.1",
     "node-sass": "^4.13.0",
-    "office-ui-fabric-react": "^7.78.0",
+    "office-ui-fabric-react": "^6.174.0",
```

Comment:
Is it still called fabric-react? Or fluent-ui?

Reply:
Why it's a downgrade?

Reply:
had fixed this issue. now is `"@fluentui/react": "^7.135.0"`